EBA/Taschenrechner/Informatikrechner/fm_Informatik.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Taschenrechner
+{
+    public partial class fm_Informatik : BaseForm
+    {
+        private readonly Zahlensystemkonvertierung Zahlensystem = new Zahlensystemkonvertierung();
+        private readonly fm_BitByte BitByte = new fm_BitByte();
+        public fm_Informatik()
+        {
+            Zahlensystem.AdviseParent += new BaseForm.AdviseParentEventHandler(SetFromForm2);
+            BitByte.AdviseParent += new BaseForm.AdviseParentEventHandler(SetFromForm2);
+            InitializeComponent();
+        }
+
+        private void btn_Zahlensystem_Click(object sender, EventArgs e)
+        {
+            Zahlensystem.Font = this.Font;
+            Zahlensystem.BackColor = this.BackColor;
+            Zahlensystem.ForeColor = this.ForeColor;
+            Zahlensystem.ShowDialog();
+        }
+        //für das Bild
+        private void btn_Bild_Click(object sender, EventArgs e)
+        {
+            ShowMessage("Bitte Bildhöhe eingeben");
+            var höhe = returnUserImputValue;
+            ShowMessage("Bitte Bildbreite eingeben");
+            var breite = returnUserImputValue;
+            ShowMessage("Bitte Farbtiefe eingeben");
+            var tiefe = returnUserImputValue;
+            var bild = höhe * breite * tiefe;
+            var rückgabe= "Ein Bild mit der Größe " + höhe + " x " + breite + " und einer Farbtiefe von " + tiefe + " hat eine Größe von " + bild + "bit";
+            SetResultInParent(rückgabe);
+        }
+
+        private void btn_Video_Click(object sender, EventArgs e)
+        {
+            ShowMessage("Bitte Bildhöhe eingeben");
+            var höhe = returnUserImputValue;
+            ShowMessage("Bitte Bildbreite eingeben");
+            var breite = returnUserImputValue;
+            ShowMessage("Bitte Farbtiefe eingeben");
+            var tiefe = returnUserImputValue;
+            ShowMessage("Bitte Länge des Videos in Sec. angeben");
+            var länge = 200; 
+            var video = höhe * breite * tiefe * länge;
```

Comment:
tha's wrong. The `länger` must be multiply by the video framerate. NTSC are 30 frames per second and PAL are 25.

Reply:
there is no given framerate in our task list.
We don't know what is required